rackcity/models/log.py

```diff
@@ -0,0 +1,30 @@
+from .it_model import ITModel
+from django.contrib.auth.models import User
+from django.db import models
+from .asset import Asset
+
+
+class Log(models.Model):
+    date = models.DateTimeField(
+        auto_now=True,
+        blank=True,
+        editable=False,
+    )
+    log_content = models.TextField()
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
```

Comment:
I don't think you want `on_delete=models.CASCADE,` here - that means if the user, model, or asset a log references is deleted, all logs about it will be deleted. You probably want either `DO_NOTHING` or `SET_NULL`

Reply:
Wondering if we use this correctly everywhere else 🤔 there's also a `PROTECT ` option which won't let you delete the referenced object until you delete the objects that reference it.  This could have been nice on the model and rack fields of Asset

Reply:
my GOD I did not actually know that's what CASCADE meant!!!!!! changing

Reply:
I did not either but stumbled upon that this morning with the port tables. Crazy stuff